README-fr.adoc

```diff
@@ -0,0 +1,310 @@
+= Asciidoctor
+Dan Allen <https://github.com/mojavelinux[@mojavelinux]>; Sarah White <https://github.com/graphitefriction[@graphitefriction]>; Ryan Waldron <https://github.com/erebor[@erebor]>
+
+include::_settings-README.adoc[]
+
+{uri-project}/[Asciidoctor] est un processeur de texte _rapide_ et une chaîne de publication pour convertir du contenu {uri-what-is-asciidoc}[AsciiDoc] en HTML5, DocBook 5 (ou 4.5) et d'autres formats.
+Asciidoctor est écrit en Ruby, packagé sous forme de RubyGem et publié sur {uri-rubygem}[RubyGems.org].
+La gem est aussi incluse dans plusieurs distributions Linux, incluant Fedora, Debian et Ubuntu.
+Asciidoctor est open source, {uri-repo}[hébergé sur GitHub] et distribué sous {uri-license}[la licence MIT].
+
+.Documentation clée
+[.compact]
+* {uri-docs}/what-is-asciidoc[Qu'est ce qu'Asciidoc ?]
+* {uri-docs}/asciidoc-writers-guide[Guide pour Ecrivain AsciiDoc]
```

Comment:
IMO "Écrivain" in French is associated to book writer, maybe "Rédacteur" de contenus ?